src/mlpack/methods/kmeans/kmeans_main.cpp

```diff
@@ -131,43 +128,36 @@ PARAM_INT_IN("max_iterations", "Maximum number of iterations before k-means "
 PARAM_INT_IN("seed", "Random seed.  If 0, 'std::time(NULL)' is used.", "s", 0);
 PARAM_MATRIX_IN("initial_centroids", "Start with the specified initial "
     "centroids.", "I");
-
 // Parameters for "refined start" k-means.
 PARAM_FLAG("refined_start", "Use the refined initial point strategy by Bradley "
     "and Fayyad to choose initial points.", "r");
 PARAM_INT_IN("samplings", "Number of samplings to perform for refined start "
     "(use when --refined_start is specified).", "S", 100);
 PARAM_DOUBLE_IN("percentage", "Percentage of dataset to use for each refined "
     "start sampling (use when --refined_start is specified).", "p", 0.02);
-
 PARAM_STRING_IN("algorithm", "Algorithm to use for the Lloyd iteration "
     "('naive', 'pelleg-moore', 'elkan', 'hamerly', 'dualtree', or "
     "'dualtree-covertree').", "a", "naive");
-
 // Given the type of initial partition policy, figure out the empty cluster
 // policy and run k-means.
 template<typename InitialPartitionPolicy>
 void FindEmptyClusterPolicy(const InitialPartitionPolicy& ipp);
-
 // Given the initial partitionining policy and empty cluster policy, figure out
 // the Lloyd iteration step type and run k-means.
 template<typename InitialPartitionPolicy, typename EmptyClusterPolicy>
 void FindLloydStepType(const InitialPartitionPolicy& ipp);
-
 // Given the template parameters, sanitize/load input and run k-means.
 template<typename InitialPartitionPolicy,
          typename EmptyClusterPolicy,
          template<class, class> class LloydStepType>
 void RunKMeans(const InitialPartitionPolicy& ipp);
-
```

Comment:
Actually all these spacing lines were here to make the code split into logical blocks.  Can we leave the blank lines there please?